### PR TITLE
Extracting date/time formats out and fixing SD-4103 SDPA-146

### PR DESCRIPTION
**Pull request by naomiaro**
porting this over to the new client-core

### CODE REVIEW

File: scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -1662,9 +1662,21 @@
                 scope.destination_last = null;
                 var PREFERENCE_KEY = 'destination:active';
 
+                scope.modelTimeFormat = 'HH:mm:ss';
+                scope.viewTimeFormat = config.view.timeformat;
+
                 scope.$watch('item', activateItem);
                 scope.$watch(send.getConfig, activateConfig);
 
+                scope.$watch('item.embargo_date', function(newVal, oldVal) {
```
Comment (akintolga) [inline scripts/superdesk-authoring/authoring.js]: There is already a watch for here: https://github.com/superdesk/superdesk-client-core/blob/master/scripts/superdesk-authoring/metadata/metadata.js#L134

Comment (naomiaro) [inline scripts/superdesk-authoring/authoring.js]: cool thanks, totally missed that.

Comment (naomiaro) [inline scripts/superdesk-authoring/authoring.js]: hmm I also see here though hardcoded date/time formats that I might need to update to work for multiple clients

https://github.com/superdesk/superdesk-client-core/blob/master/scripts/superdesk-authoring/metadata/metadata.js#L167

**Comment by naomiaro** [comment]
This pull request will also work towards fixing 
https://dev.sourcefabric.org/browse/SD-4103

**Comment by ioanpocol** [comment]
@naomiaro Please make the PR on master too

**Comment by naomiaro** [comment]
sure, will do.

**Comment by naomiaro** [comment]
This commit has changed the time format of scheduled template created at times and ingest routing schedule times in the client. A couple updates in python will need to be made to deal with this to keep time formats consistent.

**Comment by naomiaro** [comment]
Date/Time pickers have been updated to work with the new timezone directive. Time picker alt has been deleted and replaced with the general time picker. The date/time pickers receive their model and view time formats from the app config object.

**Comment by vied12** [comment]
:+1:

File: tasks/options/template.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ module.exports = function(grunt) {
         var server = grunt.option('server') || process.env.SUPERDESK_URL || url;
         var ws = grunt.option('ws') || process.env.SUPERDESK_WS_URL || 'ws://localhost:5100';
         var disableEditorToolbar = grunt.option('disableEditorToolbar');
-        var defaultTimezone = grunt.option('defaultTimezone');
+        var defaultTimezone = grunt.option('defaultTimezone') || 'Europe/London';
```
Comment (petrjasek) [inline tasks/options/template.js]: imo this could be `Europe/Prague` :smile:

**Comment by petrjasek** [comment]
:+1:

File: tasks/options/template.js (hunk 2 above)
Comment (naomiaro) [inline tasks/options/template.js]: :beers:

**Comment by akintolga** [comment]
:+1:

**Comment by akintolga** [comment]
Looks pretty good so far. One more thing to check would be the metadata preview for publish_schedule and embargo fields: Please take a look at this file: metadata-view.html. There might still be some localization going on for those fields.

**Comment by naomiaro** [comment]
Well this is annoying, https://github.com/isaacs/github/issues/18

will close and reopen a newly rebased version to master :D, and then hopefully link this pull request or something to keep the discussion track.

**Pull request closed without merging**